common/djangoapps/student/tests/tests.py

```diff
@@ -819,38 +819,6 @@ def test_change_to_honor_if_verified_not_active(self):
         self.assertEqual(enrollment_mode, u'honor')
 
 
-class PaidRegistrationTest(ModuleStoreTestCase):
-    """
-    Tests for paid registration functionality (not verified student), involves shoppingcart
-    """
-    def setUp(self):
-        super(PaidRegistrationTest, self).setUp()
-        # Create course
-        self.course = CourseFactory.create()
-        self.req_factory = RequestFactory()
-        self.user = User.objects.create(username="jack", email="jack@fake.edx.org")
-
-    @unittest.skipUnless(settings.FEATURES.get('ENABLE_SHOPPING_CART'), "Shopping Cart not enabled in settings")
-    def test_change_enrollment_add_to_cart(self):
```

Comment:
Actually, I think you should replace this test. There are two `change_enrollment` methods, one in `student/views.py` and another in `third_party_auth/pipeline.py`. You've removed the `change_enrollment` method from `third_party_auth/pipeline.py`, but this test relies on the method from `student/views.py`.